### PR TITLE
New config setting for libspecs to never prune

### PR DESCRIPTION
**Pull request by expez**
Frontend change matching https://github.com/clojure-emacs/refactor-nrepl/pull/157

### CODE REVIEW

File: clj-refactor.el [hunk 1]
```diff
@@ -221,6 +221,15 @@ won't run if there is a broken namespace in the project."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-ignore-libspecs
```
Comment (expez) [inline clj-refactor.el]: I'm quite unhappy with this name.  What do you think @benedekfazekas?

Comment (benedekfazekas) [inline clj-refactor.el]: thinking along replacing `libspecs` with `ns` but keeping `ignore` but don't have a really good one. perhaps `cljr-ns-to-ignore-patterns`?!

Comment (expez) [inline clj-refactor.el]: an ns is something else, a libspec may or may not contain a reference to a namespace.

The problem I'm having with the current name ist hat we're not really ignoring them we're just not pruning the occurrences :/

Comment (benedekfazekas) [inline clj-refactor.el]: hm.. I had not have a proper look but we might want to use this list to ignore stuff for other features as well perhaps?! if so this name is fine in terms of being generic enough

Comment (expez) [inline clj-refactor.el]: > we might want to use this list to ignore stuff for other features as well perhaps

I don't like complecting things like that :P

Comment (expez) [inline clj-refactor.el]: @benedekfazekas how about `cljr-libspec-whitelist`?

**Comment by benedekfazekas** [comment]
ok with the name 👍

**Pull request merged**